GoogleForCreators/web-stories-wp
# Background Audio: Do not always loop audio in the editor

## Context



## Summary

Ensure that the looping checkbox is respected by audio player. Ensure that play / pause button is reset when the audio player is finished. 

This change, also improves browser compatibility, as browsers like chrome allow for audio to be paused via media control or by media buttons on the keyboard.  

## Relevant Technical Choices



## To-do



## User-facing changes

https://user-images.githubusercontent.com/237508/159926210-88ee5f41-4cbf-4f31-8cb3-d9d0bee499c5.mov


## Testing Instructions






This PR can be tested by following these steps:

1. Create a story
2. Add background audio to a page.
3. Set looping to disabled.
4. Play audio. 
5. When audio is finished, it should not loop and play / pause button should reset. 

#### Media controls
1. Create a story
2. Add background audio to a page.
3. Set looping to disabled.
4. Play audio. 
5. Use media controls in Chrome to pause audio. 
6. Play / pause button should reset. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11046
